tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -189,6 +189,9 @@
         "n": "{{ jwt_rsa_key.n|long_to_base64 }}",
         "p": "{{ jwt_rsa_key.p|long_to_base64 }}",
         "q": "{{ jwt_rsa_key.q|long_to_base64 }}",
+        "dq": "{{ jwt_rsa_key.dq|long_to_base64 }}",
+        "dp": "{{ jwt_rsa_key.dp|long_to_base64 }}",
+        "qi": "{{ jwt_rsa_key.invq|long_to_base64 }}",
```

Comment:
Some notes on confirming that this is correct:

RSA private numbers are represented with slightly different names in different libraries. n, e, d, p, and q always seem to have the same names. [dp and dq aren't documented](https://www.pycryptodome.org/src/public_key/rsa#Crypto.PublicKey.RSA.RsaKey) in pycryptodome (which is what Tutor is using here) but apparently they're present and have a matching definition (d mod (p - 1) and likewise for q.) For qi we can compare with PyJWT's [translation from the `cryptography` library](https://github.com/jpadilla/pyjwt/blob/777efa2f51249f63b0f95804230117723eca5d09/jwt/algorithms.py#L316) and see that cryptography's `iqmp` has the [same definition](https://cryptography.io/en/latest/hazmat/primitives/asymmetric/rsa/#cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateNumbers) as PyCryptodome's `invq`.

(The JWK spec doesn't actually define what `qi` is, but instead refers the reader to several print books.)